init.lua

```diff
@@ -742,28 +763,49 @@ local craftable_setting = minetest.settings:get_bool("personal_log_craftable_ite
 
 if craftable_setting or not (unified_inventory_modpath or sfinv_modpath or sfinv_buttons_modpath) then
 
-minetest.register_craftitem("personal_log:book", {
-	description = S("Personal Log"),
-	inventory_image = "personal_log_open_book.png",
-	groups = {book = 1, flammable = 3},
-	on_use = function(itemstack, user, pointed_thing)
-		local name = user:get_player_name()
-		minetest.show_formspec(name,"personal_log:root", make_personal_log_formspec(user))
-	end,
-})
+	local attributes = {
+		description = S("Personal Log"),
+		inventory_image = "personal_log_open_book.png",
+		groups = {book = 1, flammable = 3},
+	}
 
-minetest.register_craft({
-	output = "personal_log:book",
-	recipe = {{book_unwritten, book_unwritten}}
-})
+	if mcl_formspec_modpath then
+		attributes.on_secondary_use = function(itemstack, user, pointed_thing)
+			local name = user:get_player_name()
+			minetest.show_formspec(name,"personal_log:root", make_personal_log_formspec(user))
+		end
+		attributes.on_place = function(itemstack, user, pointed_thing)
+			if not user:get_player_control().sneak then 
+				local new_stack = mcl_util.call_on_rightclick(itemstack, user, pointed_thing)
+				if new_stack then
+					return new_stack
+				end
+			end
+			local name = user:get_player_name()
+			minetest.show_formspec(name,"personal_log:root", make_personal_log_formspec(user))
+		end
+	else
+		attributes.on_use = function(itemstack, user, pointed_thing)
+			local name = user:get_player_name()
+			minetest.show_formspec(name,"personal_log:root", make_personal_log_formspec(user))
+		end
+	end
+
+	minetest.register_craftitem("personal_log:book", attributes)
+
+	minetest.register_craft({
+		output = "personal_log:book",
+		type = "shapeless",
+		recipe = {book_unwritten, book_unwritten}
+	})
 
 end
 
 --------------------------------------------------------------------------------------------------------
 -- Chat command
 
 local chat_command = minetest.settings:get_bool("personal_log_chat_command", false)
-local chat_command_priv = minetest.settings:get_bool("personal_log_chat_command_priviledge", false)
+local chat_command_priv = minetest.settings:get_bool("personal_log_chat_command_privilege", false)
```

Comment:
```suggestion
local chat_command_priv = minetest.settings:get_bool("personal_log_chat_command_privilege", false)
	or minetest.settings:get_bool("personal_log_chat_command_priviledge", false) -- backwards compat
```

Servers that enforce the privilege want to keep it that way. Alternatively remove the old setting from the config file and add the new one, although that's less reliable because it would only modify `minetest.conf` whereas settings can also be specified in game-specific files.